schema.json

```diff
@@ -7,7 +7,7 @@
           {
             "name": "email",
             "type": "String",
-            "isUnique": true
+            "defaultValue":"\"example@gmail.com\""  
```

Comment:
For purposes of demonstration, email can have "isUnique": `true`
We can add "defaultValue": "\"Lorem Ipsum\"" to blog title field